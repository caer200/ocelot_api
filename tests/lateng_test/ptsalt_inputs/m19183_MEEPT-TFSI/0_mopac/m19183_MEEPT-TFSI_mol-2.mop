CHARGES

 
H	8.736908 +1 12.377160 +1 5.231414 +1
C	8.717770 +1 11.895656 +1 6.050124 +1
C	9.517082 +1 10.737583 +1 6.194722 +1
C	7.914945 +1 12.335469 +1 7.073228 +1
C	9.456739 +1 10.064459 +1 7.439771 +1
N	10.315853 +1 10.302521 +1 5.143980 +1
H	7.375952 +1 13.107060 +1 6.944408 +1
C	7.875948 +1 11.666417 +1 8.304493 +1
C	8.643197 +1 10.543808 +1 8.479580 +1
S	10.423929 +1 8.691231 +1 7.823248 +1
C	11.047528 +1 9.121711 +1 5.151526 +1
C	10.515524 +1 11.252056 +1 4.015866 +1
H	7.324733 +1 11.986995 +1 9.008753 +1
H	8.624446 +1 10.086059 +1 9.311850 +1
C	11.151011 +1 8.311145 +1 6.310308 +1
C	11.724084 +1 8.684274 +1 3.990477 +1
H	10.556881 +1 12.171948 +1 4.379187 +1
H	11.387841 +1 11.058840 +1 3.589435 +1
C	9.431612 +1 11.197080 +1 2.952881 +1
C	11.921683 +1 7.140176 +1 6.297007 +1
H	11.652411 +1 9.187632 +1 3.188061 +1
C	12.484398 +1 7.539775 +1 4.007861 +1
H	9.485821 +1 12.003047 +1 2.380594 +1
H	8.541399 +1 11.185541 +1 3.385834 +1
O	9.588772 +1 10.033238 +1 2.153981 +1
H	11.980179 +1 6.606649 +1 7.080792 +1
C	12.592564 +1 6.757037 +1 5.162202 +1
H	12.945441 +1 7.275005 +1 3.220563 +1
C	8.741817 +1 10.121472 +1 1.009041 +1
H	13.124091 +1 5.969581 +1 5.160840 +1
H	7.815146 +1 10.318268 +1 1.296496 +1
H	9.047293 +1 10.861569 +1 0.426894 +1
C	8.766404 +1 8.835459 +1 0.240444 +1
H	9.702774 +1 8.549377 +1 0.093847 +1
H	8.339189 +1 8.963976 +1 -0.643369 +1
O	8.071025 +1 7.844862 +1 0.969539 +1
C	7.953492 +1 6.627400 +1 0.254520 +1
H	7.497637 +1 5.963439 +1 0.813216 +1
H	7.434214 +1 6.777704 +1 -0.563015 +1
H	8.845948 +1 6.299102 +1 0.019208 +1
