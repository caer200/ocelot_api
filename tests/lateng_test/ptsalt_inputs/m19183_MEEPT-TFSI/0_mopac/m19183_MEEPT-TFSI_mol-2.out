 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 174**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Tue Oct  8 20:08:13 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          8.73690800  *  12.37716000  *   5.23141400  *
     2       C          8.71777000  *  11.89565600  *   6.05012400  *
     3       C          9.51708200  *  10.73758300  *   6.19472200  *
     4       C          7.91494500  *  12.33546900  *   7.07322800  *
     5       C          9.45673900  *  10.06445900  *   7.43977100  *
     6       N         10.31585300  *  10.30252100  *   5.14398000  *
     7       H          7.37595200  *  13.10706000  *   6.94440800  *
     8       C          7.87594800  *  11.66641700  *   8.30449300  *
     9       C          8.64319700  *  10.54380800  *   8.47958000  *
    10       S         10.42392900  *   8.69123100  *   7.82324800  *
    11       C         11.04752800  *   9.12171100  *   5.15152600  *
    12       C         10.51552400  *  11.25205600  *   4.01586600  *
    13       H          7.32473300  *  11.98699500  *   9.00875300  *
    14       H          8.62444600  *  10.08605900  *   9.31185000  *
    15       C         11.15101100  *   8.31114500  *   6.31030800  *
    16       C         11.72408400  *   8.68427400  *   3.99047700  *
    17       H         10.55688100  *  12.17194800  *   4.37918700  *
    18       H         11.38784100  *  11.05884000  *   3.58943500  *
    19       C          9.43161200  *  11.19708000  *   2.95288100  *
    20       C         11.92168300  *   7.14017600  *   6.29700700  *
    21       H         11.65241100  *   9.18763200  *   3.18806100  *
    22       C         12.48439800  *   7.53977500  *   4.00786100  *
    23       H          9.48582100  *  12.00304700  *   2.38059400  *
    24       H          8.54139900  *  11.18554100  *   3.38583400  *
    25       O          9.58877200  *  10.03323800  *   2.15398100  *
    26       H         11.98017900  *   6.60664900  *   7.08079200  *
    27       C         12.59256400  *   6.75703700  *   5.16220200  *
    28       H         12.94544100  *   7.27500500  *   3.22056300  *
    29       C          8.74181700  *  10.12147200  *   1.00904100  *
    30       H         13.12409100  *   5.96958100  *   5.16084000  *
    31       H          7.81514600  *  10.31826800  *   1.29649600  *
    32       H          9.04729300  *  10.86156900  *   0.42689400  *
    33       C          8.76640400  *   8.83545900  *   0.24044400  *
    34       H          9.70277400  *   8.54937700  *   0.09384700  *
    35       H          8.33918900  *   8.96397600  *  -0.64336900  *
    36       O          8.07102500  *   7.84486200  *   0.96953900  *
    37       C          7.95349200  *   6.62740000  *   0.25452000  *
    38       H          7.49763700  *   5.96343900  *   0.81321600  *
    39       H          7.43421400  *   6.77770400  *  -0.56301500  *
    40       H          8.84594800  *   6.29910200  *   0.01920800  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          8.7369     12.3772      5.2314
     2         C          8.7178     11.8957      6.0501
     3         C          9.5171     10.7376      6.1947
     4         C          7.9149     12.3355      7.0732
     5         C          9.4567     10.0645      7.4398
     6         N         10.3159     10.3025      5.1440
     7         H          7.3760     13.1071      6.9444
     8         C          7.8759     11.6664      8.3045
     9         C          8.6432     10.5438      8.4796
    10         S         10.4239      8.6912      7.8232
    11         C         11.0475      9.1217      5.1515
    12         C         10.5155     11.2521      4.0159
    13         H          7.3247     11.9870      9.0088
    14         H          8.6244     10.0861      9.3118
    15         C         11.1510      8.3111      6.3103
    16         C         11.7241      8.6843      3.9905
    17         H         10.5569     12.1719      4.3792
    18         H         11.3878     11.0588      3.5894
    19         C          9.4316     11.1971      2.9529
    20         C         11.9217      7.1402      6.2970
    21         H         11.6524      9.1876      3.1881
    22         C         12.4844      7.5398      4.0079
    23         H          9.4858     12.0030      2.3806
    24         H          8.5414     11.1855      3.3858
    25         O          9.5888     10.0332      2.1540
    26         H         11.9802      6.6066      7.0808
    27         C         12.5926      6.7570      5.1622
    28         H         12.9454      7.2750      3.2206
    29         C          8.7418     10.1215      1.0090
    30         H         13.1241      5.9696      5.1608
    31         H          7.8151     10.3183      1.2965
    32         H          9.0473     10.8616      0.4269
    33         C          8.7664      8.8355      0.2404
    34         H          9.7028      8.5494      0.0938
    35         H          8.3392      8.9640     -0.6434
    36         O          8.0710      7.8449      0.9695
    37         C          7.9535      6.6274      0.2545
    38         H          7.4976      5.9634      0.8132
    39         H          7.4342      6.7777     -0.5630
    40         H          8.8459      6.2991      0.0192

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C17 H19 N O2 S  =    40 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
